### PR TITLE
feat(privacy page): Privacy Settings Step during Sign Up Flow

### PR DESCRIPTION
**Pull request by genie-magic**
**What this PR does** 📖
Privacy Settings Step during Sign Up Flow
**Which issue(s) this PR fixes** 🔨

Fixes # AP-19

**Special notes for reviewers** 🗒️
1.  When test , first clear local storage -> input pin -> create account -> privacy page
2. This needs locales change. please test this after https://github.com/Satellite-im/locales/pull/11 is merged to locales.

### CODE REVIEW

**Comment by stavares843** [comment]
merged https://github.com/Satellite-im/locales/pull/11

**Review by stavares843 — CHANGES_REQUESTED** [review]
Captura de ecrã 2021-12-10, às 18 33 34

ignore the locales, I can't use the last toggle

returns the following error

`[vuex] unknown mutation type: settings/displayCurrentActivity`

**Comment by genie-magic** [comment]
> displayCurrentActivity

Can you send me screenshot of your console. I can't see that issue. Also did you get latest code?

**Comment by stavares843** [comment]
yes i already have the latest code, when i pulled out the branch, already went with those 3 commits


but the issue is no longer happening not sure why

this was the error

Captura de ecrã 2021-12-10, às 18 33 55

**Comment by stavares843** [comment]
that error no longer happens and all toggles work fine now but I noticed on mobile, we can only see 3 toggles, and there is no scroll - note: already fixed that privacy typo on the title here - https://github.com/Satellite-im/locales/commit/63be50399e5c8b0e51b6aa83cd8be912d68cf268


Captura de ecrã 2021-12-10, às 22 20 13

**Comment by genie-magic** [comment]
> that error no longer happens and all toggles work fine now but I noticed on mobile, we can only see 3 toggles, and there is no scroll - note: already fixed that privacy typo on the title here - [Satellite-im/locales@63be503](https://github.com/Satellite-im/locales/commit/63be50399e5c8b0e51b6aa83cd8be912d68cf268)
> 
> Captura de ecrã 2021-12-10, às 22 20 13

@stavares843  I fixed issue on mobile. plz check.

**Comment by stavares843** [comment]
thanks! 🎉 checking

**Review by WanderingHogan — APPROVED** [review]
looks good

File: pages/setup/privacy/index.vue [hunk 1]
```diff
@@ -2,16 +2,48 @@
...
   data() {
     return {
       registry: true,
-      storePin: false,
-      embeds: true,
-      activity: true,
     }
   },
+  computed: {
+    ...mapState(['ui', 'accounts', 'settings']),
```
[the remark this answers is not in the document]
Comment (WanderingHogan) [reply pages/setup/privacy/index.vue:15]: this doesn't need to be updated here

**Review by Jekrimo — APPROVED** [review]

**Comment by stavares843** [comment]
mobile issue fixed 🎉

**Review by stavares843 — APPROVED** [review]

**Pull request merged**